internal/local/app.go

```diff
@@ -208,6 +214,20 @@ func (a App) WriteConfig() error {
 	return WriteFile(filepath.Join(a.RootDir, a.Config.String()), 0666, bytes.NewReader(data))
 }
 
+// WriteCliConfig writes the .mdb/app_meta.json config to the specified path given a remote app and config version
+func (a App) WriteCliConfig() error {
+	data, err := MarshalJSON(a.AppMeta)
+	if err != nil {
+		return err
+	}
+
+	if err := WriteFile(filepath.Join(a.RootDir, FileMdbAppMeta.String()), 0666, bytes.NewReader(data)); err != nil {
```

Comment:
similarly to WriteConfig(), you could just
`return WriteFile(filepath.Join(a.RootDir, FileMdbAppMeta.String()), 0666, bytes.NewReader(data))`
and skip the `if`